va.data-commons.org/portal/gitops.json

```
@@ -1,5 +1,6 @@
 {
   "gaTrackingId": "UA-119127212-10",
+  "argo_template": "gwas-template",
```

```suggestion
  "argoTemplate": "gwas-template",
```

usually we use camel cases for portal config fields

---

cool changing